submit_a_new_code.md

```diff
@@ -8,11 +8,15 @@ comment: >-
 
 ### Call for Community Codes
 
-This website is intended to help users judge for themselves the
-suitability of available codes for their purposes. Performance on the
-various benchmark problems are expected to form the major basis for
-comparison, but consideration should also be given to ease of
-installation, use, and comprehension of the code.
+This website is intended to help users make their own decisions regarding
+the suitability of the available codes for their research goals. For example,
+all codes are not equally suited to for solving all types of physics that may
+be incorporated into phase field models.  Or, a certain trade-off between
+performance and flexibility may be needed.  The numerical performance of the codes
+for the various benchmark problems will likely form the major basis for
+comparison, but other factors to consider include the ease of installation,
+learning curve to use the code, software compatiblity with different computing
+platforms, available documentation, etc.
```

Comment:
Please don't conclude with "etc.", since "factors to consider include" makes it implicit.

Reply:
interesting thought.  I'll reword to avoid conflict on this.